doc/tutorial/search/solution4.cpp

```diff
@@ -0,0 +1,37 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+
+using namespace seqan3;
+using namespace seqan3::search_cfg;
+
+int main()
+{
+    dna4_vector text{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTAACCCGATGAGCTACCCAGTAGTCGAACTGGGCCAGACAACCCGGCGCTAATGCACTCA"_dna4};
+    dna4_vector query{"GCT"_dna4};
+
+    fm_index index{text};
+
+    debug_stream << "Searching all hits\n";
+    configuration const cfg_all = search_cfg::max_error{search_cfg::total{1}}
+                                  | search_cfg::mode{search_cfg::all};
```

Comment:
Have we decided of where to put the `|` operator. AFAIK we always break after the operator and not before